app/javascript/app/components/ndc-sdg/ndc-sdg-linkages-map/ndc-sdg-linkages-map-styles.scss

```diff
@@ -1,8 +1,33 @@
 @import '~styles/layout.scss';
+@import '~styles/settings.scss';
 
 .container {
-  height: 100%;
-  position: relative;
+  position: absolute;
+  width: 100%;
+  height: 50vh;
```

Comment:
It would be nice to use only around 30-35vh for the map like in the designs, so we don't have that empty space on the top

Reply:
Agree, the design seems to be using the 40% of the height.